src/transformers/pug.ts

```diff
@@ -67,8 +67,21 @@ const transformer: Transformer<Options.Pug> = async ({
     ...options,
   };
 
+  // separate content that needs to come before the svelte mixins
+  let preContent = '';
+
+  if (content.startsWith('extends')) {
```

Comment:
Given the [example of inheritance from pug's documentation](https://pugjs.org/language/inheritance.html), what do you think of also adding support for files that start with a comment like:

```pug
//- page-a.pug
extends layout.pug

block scripts
  script(src='/jquery.js')
  script(src='/pets.js')
```